internal/testutils/feature.go

```diff
@@ -19,7 +24,11 @@ func checkFeatureTestError(t *testing.T, err error) {
 
 	var ufe *internal.UnsupportedFeatureError
 	if errors.As(err, &ufe) {
-		checkKernelVersion(t, ufe)
+		if ignoreKernelVersionCheck(t) {
+			t.Skip(ufe.Error())
```

Comment:
Maybe add some more context here? Otherwise it's not clear why we skipped that test.

```suggestion
			t.Skip("Ignoring error due to EBPF_TEST_IGNORE_KERNEL_VERSION:", ufe.Error())
```

Reply:
I agree. I will add more context.